Description: Use cups-pk-helper in *Mechanism.conf
 .
 cups-pk-helper (0.2.6-1ubuntu4~artful1) artful; urgency=medium
 .
   * Try: prevent delete of other's jobs
Author: Shem Pasamba <shemgp@gmail.com>
Forwarded: no
Last-Update: 2017-12-20

--- cups-pk-helper-0.2.6.orig/src/org.opensuse.CupsPkHelper.Mechanism.conf
+++ cups-pk-helper-0.2.6/src/org.opensuse.CupsPkHelper.Mechanism.conf
@@ -6,7 +6,7 @@
 <busconfig>
 
   <!-- Only root can own the service -->
-  <policy user="root">
+  <policy user="cups-pk-helper">
     <allow own="org.opensuse.CupsPkHelper.Mechanism"/>
   </policy>
 
